src/provider/treeView.ts

```diff
@@ -160,10 +160,25 @@ export class MaestroWorkBenchTreeViewProvider implements vscode.TreeDataProvider
                                 const dependencyPath = path.resolve(path.dirname(filePath), flow.runFlow.file);
                                 dependencies.add(dependencyPath);
                             }
+
                             if (flow.runScript && flow.runScript.file) {
                                 const dependencyPath = path.resolve(path.dirname(filePath), flow.runScript.file);
                                 dependencies.add(dependencyPath);
                             }
+
+                            if (flow.addMedia) {
```

Comment:
Does this miss any addMedia commands that are in onFlowStart / onFlowComplete / runFlow / repeat (possibly other) commands?